src/index.js

```diff
@@ -0,0 +1,25 @@
+const { exec } = require('child_process')
+
+module.exports = function (pluginConfig, config, cb) {
+  exec('git tag', (err, stdout, stderr) => {
+    if (err) return cb(err)
+
+    const versionRegex = /^v((?:[0-9]+\.){2}[0-9]+)$/
```

Comment:
This is not catching all valid semver versions. I'd suggest using a library like `semver` or at least `semver-regex` for this.